curriculum/challenges/english/10-coding-interview-prep/rosetta-code/josephus-problem.md

```diff
@@ -86,36 +86,13 @@ function josephus(init, kill) {
 
 ```js
 function josephus(init, kill) {
-  var Josephus = {
-    init: function(n) {
-      this.head = {};
-      var current = this.head;
-      for (var i = 0; i < n - 1; i++) {
-        current.label = i + 1;
-        current.next = {
-          prev: current
-        };
-        current = current.next;
-      }
-      current.label = n;
-      current.next = this.head;
-      this.head.prev = current;
-      return this;
-    },
-    kill: function(spacing) {
-      var current = this.head;
-      while (current.next !== current) {
-        for (var i = 0; i < spacing - 1; i++) {
-          current = current.next;
-        }
-        current.prev.next = current.next;
-        current.next.prev = current.prev;
-        current = current.next;
-      }
-      return current.label;
-    }
+  const arr = Array.from(Array(init).keys());
+  let curr = -1
+  while (arr.length > 1) {
+    curr = (curr+kill)%arr.length;
```

Comment:
Just a small nitpicking to put spaces around `+` and `%` here.

Reply:
```suggestion
    curr = (curr + kill) % arr.length;
```
A committable suggestion, as requested in our Discord Chat. @alirezaghey to commit this you can use the UI and click "Commit changes", create your commit message, and submit it. 🙂